www/index.html

```diff
@@ -906,6 +917,41 @@
           </div>
         {{/each}}
       </script>
+
+      <script type="text/template7" id="galleryTemplate">
+        {{#each album}}
+          <a href="album/{{id}}">
+            <div class="albums">
+              <div class="album-full-background" style="background-image: url({{thumb}})"></div>
+              <div class="album-text">
+                <div class="title"><p> {{title}} </p></div>
+                <div><i class="fa fa-calendar"></i> {{js "new Date(this.start_date).shortDateString()"}}</div>
+                <div><i class="fa fa-map-marker"></i> {{location}}</div>
+                <div><i class="fa fa-image"></i> {{image_count}}</div>
+              </div>
+            </div>
+          </a>
+        {{/each almbum}}
+      </script>
+
+      <script type="text/template7" id="albumTemplate">
+        <div class="block-title">{{title}}</div>
+        <div class="album-info-block">
+          {{#if description}}
+            <p>{{description}}</p>
+          {{/if}}
+          {{#if photographers}}
+            <p><span>Fotografer: </span>{{join photographers delimiter=", "}}</p>
+          {{/if}}
+        </div>
+        {{#each image}}
+        <a href="#" data-index="{{@index}}" class="album-imageBrowser">
+          <div class="album-images">
```

Comment:
You should have the each loop inside a container, called `album-images` or something, to have a better structure in your page. Also, we don't have camel case names for classes in css. Use `album-image-browser` instead and change to singular `album-image` just as in gallery.